salt/modules/junos.py

```diff
@@ -794,33 +801,25 @@ def install_config(path=None, **kwargs):
         write_diff = op['diffs_file']
         del op['diffs_file']
 
-    if 'template_path' in op:
-        if not os.path.isfile(op['template_path']):
-            ret['message'] = 'Invlaid template path.'
-            ret['out'] = False
-            return ret
-        if 'template_vars' not in op:
-            ret[
-                'message'] = 'Please provide jinja variable along with the \
-                jina template.'
-            ret['out'] = False
-            return ret
-        if not os.path.isfile(op['template_vars']):
-            ret['message'] = 'Invlaid template_vars path.'
-            ret['out'] = False
-            return ret
-        data = yaml.load(open(op['template_vars']))
-        op['template_vars'] = data
-    else:
-        op = {'path': path}
+    op['path'] = template_cached_path
+
+    if 'format' not in op:
```

Comment:
we can skip format setting checks as PyEZ takes care of this. Its always better to have such check with PyEZ itself. So that any tools be it Ansible salt or anything in future can save single source.
Say we also support json for > 14.x junos, which not handled here. 
refer
https://github.com/Juniper/py-junos-eznc/blob/master/lib/jnpr/junos/utils/config.py#L306-L315